content/sensu-go/5.18/release-notes.md

```diff
@@ -53,6 +54,28 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.18.0 release notes
+
+**February 24, 2020** &mdash; The latest release of Sensu Go, version 5.18.0, is now available for download.
+**PLACEHOLDER FOR RELEASE SUMMARY**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.18.0.
+
+**IMPROVEMENTS:**
+
+- The `event.entity.entity_class` value now defaults to `proxy` for [`POST /events`][114] requests.
+- Proxy entities are now automatically created when events are published with an entity that does not exist.
+- Supported Bonsai asset versions are now prefixed with the letter `v` (for example, `v1.2.0`).
+- The version API now retrieves the Sensu agent version for the Sensu instance.
```

Comment:
Just to make sure it's clear to everyone, this new version API was added to sensu-agent (`http://127.0.0.1:3031/version`), so it's different from https://docs.sensu.io/sensu-go/latest/api/version/!

Reply:
OK -- so we should not document it in the API section, but we do need to add it in the agent reference somewhere...does that sound right?

Reply:
Yep!